perun-base/src/main/java/cz/metacentrum/perun/core/api/RoleAssigmentType.java

```diff
@@ -0,0 +1,6 @@
+package cz.metacentrum.perun.core.api;
+
+public class RoleAssigmentType {
```

Comment:
Why not make this an enum? And also use it everywhere instead of `string` (would eliminate extra code and make it more readable)

Reply:
@sarkapalkovicova